core/src/main/java/org/infinispan/transaction/xa/recovery/InDoubtTxInfoImpl.java

```diff
@@ -22,28 +21,18 @@
 public class InDoubtTxInfoImpl implements RecoveryManager.InDoubtTxInfo {
    private Xid xid;
    private Long internalId;
```

Comment:
can you convert to `long`
```suggestion
   private long internalId;
```